### PR TITLE
Script updated with wireframe

### PR DESCRIPTION
**Pull request by avishekjana**
#62

### CODE REVIEW

File: making-api-requests/intro-to-api-service/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+![screen-task](screen-task.png)
+
+In that dialog, the user should be able to edit the task title, description, due date and assignee name, using a feature called *in-place editing*.
+
+After these basic task details, the user should get an option to add comment for that task. And just below that, he/she should be able to see the list of all comments.
+
+Then from the main menu, the user should be able to visit and see the list of all members in that organisation, and he/she can invite others to join as well.
+
+### Understanding the API service
+Now, to implement all these features, we need a lot of APIs. But, don't worry, we've created each and every API endpoints required for this project, and it's deployed at:
```
Comment (reenas) [inline making-api-requests/intro-to-api-service/README.md:54]: @avishekjana API word is being used for the first time here. We should explain what are they and what purpose do they have, with some real-world examples if possible.

File: making-api-requests/intro-to-api-service/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+1. Add a new task
+2. Update a task
+3. Delete a task
+4. Get list of all tasks
+5. Add a new comment against a task
+6. Get the list of all comments made against a task
+
+So, please go through the API doc, on your own. There you'll notice, we've added a lot of details in terms of, what are the parameters, which ones are required, sample request body, sample response body and all possible HTTP status codes you might encounter based on client or server errors.
+
+Now, most of these endpoints are secured, except the endpoints for *creating new organisation* and *user signin*. So, in all of these secured endpoints, you've to send a token as part of a request header called **Authorization**. You'll get the token once you would signin to the application. In our React app, we've to make sure to store that token in the browser and send it as part of the  **Authorization** header, to call the protected endpoints.
```
Comment (reenas) [inline making-api-requests/intro-to-api-service/README.md:91]: @avishekjana Mention why is it important to secure an endpoint and what might happen if its not done.

File: making-api-requests/intro-to-api-service/README.md (hunk 1 above)
Comment (avishekjana) [reply making-api-requests/intro-to-api-service/README.md:54]: OK @reenas , I'll add, but students already learned about APIs in WD201.

Comment (avishekjana) [reply making-api-requests/intro-to-api-service/README.md]: I've updated the script with a short intro about API.

**Pull request merged**